EIPS/eip-7805.md

```diff
@@ -95,11 +95,11 @@ For each transaction `T` in ILs, perform the following:
 
 1. Check whether `T` is present in `B`. If `T` is present, then jump to the next transaction, else continue with next step.
 
-2. Validate `T` against `S`. 
+2. Validate `T` against `S`.
 
-  - If `T` is invalid, then continue to the next transaction. 
-  
-  - If `T` is valid, terminate process and assert block `B` as invalid.
+   - If `T` is invalid, then continue to the next transaction.
+
+   - If `T` is valid, terminate process and assert block `B` as invalid.
 
 3. Execute `T` on state `S`. Assert that the execution of `T` fails.
```

Comment:
BTW, under which 2. sub-bullet do we end up in step 3.?

Shouldn't the second sub-bullet say moving to the next step (i.e. 3.) and do the assertion?